packages/lodestar/src/constants/network.ts

```diff
@@ -38,9 +41,14 @@ export type RpcResponseStatusError = Exclude<RespStatus, RespStatus.SUCCESS>;
 
 /** The maximum allowed size of uncompressed gossip messages. */
 export const GOSSIP_MAX_SIZE = 2 ** 20;
+
+// source: https://github.com/ethereum/eth2.0-specs/blob/f52f067b8ea3f8adbebc936207b06459d1956e72/specs/phase0/p2p-interface.md#why-are-blocksbyrange-requests-only-required-to-be-served-for-the-latest-min_epochs_for_block_requests-epochs
+export function getMinEpochForBlockRequests(config: IChainForkConfig): number {
+  return config.MIN_VALIDATOR_WITHDRAWABILITY_DELAY + intDiv(config.CHURN_LIMIT_QUOTIENT, 2);
```

Comment:
Is this formula right? The link above states
```
MIN_VALIDATOR_WITHDRAWABILITY_DELAY
    + MAX_SAFETY_DECAY * CHURN_LIMIT_QUOTIENT // (2 * 100)
```

Reply:
Seems like same thing, since MAX_SAFETY_DECAY is 100. Probably added later to ease in changing MAX_SAFETY_DECAY. Pushing fix